app/frontend/src/submissions/views/SubmissionsHome.vue

```diff
@@ -579,9 +582,115 @@ export default {
       }
       this.componentUpdateTrigger = Date.now()
     },
+    groundwaterProtectionRegulationValidation(errors) {
+      const {
+        owner_full_name, 
+        owner_mailing_address,
+        owner_city,
+        owner_province_state,
+        owner_postal_code,
+      } = this.form
+
+      if (!owner_full_name) {
+        errors.owner_full_name = ['Owners Full Name Required.'];
+      }
+      if (!owner_mailing_address) {
+        errors.owner_mailing_address = ['Owners Mailing Address Required.'];
+      }
+      if (!owner_city) {
+        errors.owner_city = ['Owners City Required.'];
+      }
+      if (!owner_province_state) {
+        errors.owner_province_state = ['Owners Province or State Required.'];
+      }
+      if (!owner_postal_code) {
+        errors.owner_postal_code = ['Owners Postal Code Required.'];
+      }
+      if (owner_postal_code && !isValidPostalCodeOrZipCode(owner_postal_code)) {
+        errors.owner_postal_code = ['Invalid Postal Code or ZIP Code.'];
```

Comment:
```suggestion
        errors.owner_postal_code = ['Invalid Postal Code or ZIP Code. e.g. V2A 2A2'];
```
Since the whitespace character is enforced, we should add an example for clarity 
![image](https://github.com/bcgov/gwells/assets/62873746/2ac41ab5-e558-4a40-9540-b1e6348cd003)

Reply:
I am going to update the validation to allow the space.